docs/libp2p-resource-management.md

```diff
@@ -1,28 +1,52 @@
+# libp2p Network Resource Manager <small>(`Swarm.ResourceMgr`)</small>
+
 ## Purpose
 The purpose of this document is to provide more information about the [libp2p Network Resource Manager](https://github.com/libp2p/go-libp2p/tree/master/p2p/host/resource-manager#readme) and how it's integrated into Kubo so that Kubo users can understand and configure it appropriately.
 
-## 🙋 Help!  The resource manager is protecting my node but I want to understand more 
-The resource manager is generally a *feature* to bound libp2p's resources, whether from bugs, unintentinoally misbehaving peers, or intentional Denial of Service attacks.
+## 🙋 Help!  The resource manager is protecting my node but I want to understand more
+The resource manager is generally a *feature* to bound libp2p's resources, whether from bugs, unintentionally misbehaving peers, or intentional Denial of Service attacks.
 
 Good places to start are:
 1. Understand [how the resource manager is configured](#levels-of-configuration).
 2. Understand [how to read the log message](#what-do-these-protected-from-exceeding-resource-limits-log-messages-mean)
-3. Confirm [the log message is from your local node](#what-are-the-application-error--cannot-reserve--messages)
-4. [Monitor resource usage](#how-does-one-monitor-libp2p-resource-usage) and compare them to the [active limits](#how-does-one-see-the-active-limits) with the  
+3. Understand [how to inspect and change limits](#user-supplied-override-limits)
+
+## Table of Contents
```

Comment:
FAQ nature of this document makes Table of contents useful for quick scanning, but we can remove it if the maintenance becomes too much.